src/app/(app)/missing-shoes/add/components/MissingShoesActions.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+
+import React, { useMemo } from "react";
+import ActionsMenu from "@/common/ActionMenu/ActionsMenu";
+import {
+  Trash2 as TrashIcon,
+  Edit as EditIcon,
+  CheckCircle as CheckCircleIcon,
+  AlertCircle as AlertCircleIcon,
+} from "lucide-react";
+import { MissingShoeStatus } from "@/constant/constant";
+
+export const MissingShoesActions = ({
+  missingShoesId,
+}: {
+  missingShoesId: number;
+}) => {
+  const handleEditInfo = (id: number) => {
+    return id;
+  };
+
+  const handleDeleteRequest = async (id: number) => {
+    return id;
+  };
+
+  const handleLeaveRequestStatus = async (
+    id: number,
+    status: MissingShoeStatus
+  ) => {
+    return {
+      id,
+      status,
+    };
+  };
+
+  const actionMenu = useMemo(
+    () => [
+      {
+        title: "Found",
+        id: 4,
+        onClick: () => {
+          handleLeaveRequestStatus(missingShoesId, MissingShoeStatus.Found);
+        },
+        icon: <CheckCircleIcon size={16} />,
+      },
+      {
+        title: "Missing",
+        id: 4,
+        onClick: () => {
+          handleLeaveRequestStatus(missingShoesId, MissingShoeStatus.Missing);
+        },
+        icon: <AlertCircleIcon size={16} className="stroke-status-inactive"/>,
+      },
+      {
+        title: "Edit",
+        id: 2,
+        onClick: () => {
+          handleEditInfo(missingShoesId);
+        },
+        icon: <EditIcon size={16} />,
+      },
+      {
+        title: "Delete",
+        id: 3,
+        onClick: () => {
+          handleDeleteRequest(missingShoesId);
```

Comment:
we will not delete. just update status ,

Reply:
That's true. I'd urge you all to put your questions in #shoes-rack channel. It's easy to misunderstand the requirement.

Reply:
Sure Sir I have fixed this one